app/logic/utils.py

```diff
@@ -42,3 +43,15 @@ def getStartofCurrentAcademicYear(currentTerm):
         fallTerm = Term.select().where(Term.year==currentTerm.year-1, Term.description == f"Fall {currentTerm.year-1}").get()
         return fallTerm
     return currentTerm
+
+def format24HourTime(timeStr):
```

Comment:
To save ourselves some potential pain later, let's also handle if timeStr is a datetime object.